.github/workflows/build.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - name: Extract Clang 9
       run: tar xf clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-18.04.tar.xz && mv clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-18.04 clang9
     - name: Setup boolector
-      run: git clone https://github.com/boolector/boolector && cd boolector && ./contrib/setup-lingeling.sh && ./contrib/setup-btor2tools.sh && ./configure.sh && cd build && make -j9
+      run: git clone https://github.com/boolector/boolector && cd boolector && ./contrib/setup-lingeling.sh && ./contrib/setup-btor2tools.sh && ./configure.sh --prefix $PWD/../boolector-release && cd build && make -j9 && make install
```

Comment:
Could you force a stable boolector tag? Otherwise, we'll be building with the nightly version and it might break the compilation.

Reply:
Forced into 3.1.0